scvi/data/_anndata.py

```diff
@@ -70,6 +71,7 @@ def get_from_registry(adata: anndata.AnnData, key: str) -> np.ndarray:
     return data
 
 
+@deprecated(extra="Please use the model-specific setup_anndata methods instead.")
```

Comment:
should we denote what version we expect to remove it?

Reply:
sure, added 0.15.0